sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/HomeComposeScreen.kt

```diff
@@ -1,29 +1,64 @@
 package co.nimblehq.sample.compose.ui.screens.home
 
-import androidx.compose.foundation.layout.Box
-import androidx.compose.foundation.layout.fillMaxSize
+import android.widget.Toast
+import androidx.compose.foundation.layout.*
 import androidx.compose.material.CircularProgressIndicator
 import androidx.compose.material.Scaffold
-import androidx.compose.runtime.Composable
+import androidx.compose.runtime.*
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.hilt.navigation.compose.hiltViewModel
 import co.nimblehq.sample.compose.R
+import co.nimblehq.sample.compose.lib.IsLoading
 import co.nimblehq.sample.compose.model.UiModel
+import co.nimblehq.sample.compose.ui.AppDestination
 import co.nimblehq.sample.compose.ui.screens.AppBar
-import co.nimblehq.sample.compose.ui.theme.ComposeSampleTheme
+import co.nimblehq.sample.compose.ui.theme.ComposeTheme
+import co.nimblehq.sample.compose.ui.userReadableMessage
 
 // TODO: Rename to 'HomeScreen'
 @Composable
 fun HomeComposeScreen(
+    viewModel: HomeComposeViewModel = hiltViewModel(),
```

Comment:
Shall we rename this ViewModel too?
```suggestion
    viewModel: HomeViewModel = hiltViewModel(),
```

Reply:
Same, fixed in https://github.com/nimblehq/android-templates/pull/372